### ISSUE TITLE
Discovery: Accessible design pattern for CMS table row hover states

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9917]
## Background

An accessibility audit found that on all products and all tables, the main content listing has tables that do not pass WCAG accessibility standards for hover states. Table rows currently have a hover state that changes to light blue. 

On hover, the background color of the table rows changes to light blue. However, because of this, the color contrast between elements no longer passes:
1. Between the links and the background
2. And between the backgound and the link hover background color

Image of defective area:
<img width="276" alt="Screen Shot 2022-07-27 at 12 36 47 PM" src="https://user-images.githubusercontent.com/90111970/181312854-a06cd524-cf2a-469f-9b4d-f8f5c6d15252.png">

## Possible Solution

There does not have to be a hover state on the table row, this can be removed if desired.  Or if it should stay for design reasons, the color should be changed to make sure it passes ratio requirements with the standard link color and the link hover background color

## Accessibility Standard

WCAG version 2.0 AA, [Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

## Acceptance Criteria

- [x] Discovery of table usage across products to determine if the hover is necessary for design or other reasons not immediately obvious in audit
- [x] WCAG AAA considered
- [x] Hover state pattern with appropriate color contrast for link color
- [x] Design is reviewed with Accessibility Lead
- [x] Document changes to links in the CMS DS and changelog
- [x] tech implementation ticket created

**Comment by EWashb** [comment]
After fixing this issue, additional tickets for hover state color contrast on links across the CMS?

**Comment by EWashb** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VAgov-CMS-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/9917?planning-poker) with ZenHub @cjterdi @laflannery

**Comment by cjterdi** [comment]
Per our accessibility review for links, we have come up with the follow criteria. This applies to [9916](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9916) and [9915](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9915)

All standard links will be changed to the dark blue (004795)
hover/focus state link color will be darker blue (003e73)
We think having a background color would be beneficial as this would keep it as a similar format as the front end (they have a background color on hover)
The hover/focus state should be the lighter blue (e4eff9). This is currently the color on some CMS links, not all. It is also different than the VA.gov front end hover color
Footer links will remain white
Footer links on hover/focus will keep their current background color (e4eff9) but the text will be darker blue (003e73)
All links should have underline on default and then no underline on hover/focus

**Comment by EWashb** [comment]
Implementation ticket for these changes has been created.
